yarn-project/end-to-end/src/fixtures/snapshot_manager.ts

```diff
@@ -291,6 +297,17 @@ async function setupFromFresh(
   } else {
     aztecNodeConfig.dataDirectory = statePath;
   }
+  aztecNodeConfig.blobSinkUrl = `http://127.0.0.1:${blobSinkPort}`;
```

Comment:
This and the synching test uses localhost vs 127.0.0.1, should we be more consistent here?